components/raftstore/src/store/fsm/peer.rs

```diff
@@ -983,6 +984,37 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
             self.fsm.has_ready = true;
         }
 
+        // If lease expired, we will send a noop read index to renew lease.
+        if self.fsm.peer.is_leader() {
+            let current_time = match self.ctx.current_time {
+                None => {
+                    self.ctx.current_time = Some(monotonic_raw_now());
+                    self.ctx.current_time.unwrap()
+                }
+                Some(t) => t,
+            };
+            let next_tick = current_time + self.ctx.cfg.raft_base_tick_interval();
```

Comment:
`current_time` may older than `monotonic_raw_now` and we may wait longer than `raft_base_tick_interval` to trigeer next `on_raft_base_tick` on this peer, so should we times the `raft_base_tick_interval` here? i.e: `let next_tick = current_time + self.ctx.cfg.raft_base_tick_interval() * 2;` @5kbpers @overvenus @nolouch

Reply:
I think `let next_tick = current_time + self.ctx.cfg.raft_base_tick_interval();` is fine.